app/_src/gateway/reference/configuration.md

```diff
@@ -4186,12 +4186,37 @@ and valid UUID. When empty, node ID is automatically generated.
 
 **Default:** none
 
+---
+
+## Cluster Fallback Configuration section
+
+
+### cluster_fallback_config_export
```

Comment:
Based on https://github.com/Kong/kong-ee/blob/master/kong.conf.default#L3388-L3390, these entries should come after the other two in the list.

Reply:
Ah I see, I was trying to make it alphabetical, I didn't fully realize it had to match the order from kong.conf, but that makes sense to me then. 👍

Reply:
Made the correction now.